Allow running crrExistingObjects with sharded mongo

francoisferrand opened the pull request:
- Move createMongoParams() to utils to make it reusable
- Use createMongoParams() in CrrExistingObjects script
- Use createMongoParams() in stalled script

Issue: S3UTILS-76

francoisferrand committed:
Move createMongoParams() to utils

This way it can be used in other scripts.

Issue: S3UTILS-76

bert-e commented:
# Hello francoisferrand,



My role is to assist you with the merge of this
pull request. Please type `@bert-e help` to get information
on this process, or consult the [user documentation](
https://eve.devsca.com/github/scality/s3utils/bert-e/doc/user).





*Status report is not available.*

bert-e commented:
# Incorrect fix version



The `Fix Version/s` in issue S3UTILS-76 contains:


* `1.12.6`



Considering where you are trying to merge, I ignored possible hotfix versions and I expected to find:



* `1.12.6`

* `1.13.0`


Please check the `Fix Version/s` of S3UTILS-76, or the target
branch of this pull request.

francoisferrand committed:
Use createMongoParams() in CrrExistingObjects script

This way, the script can be used with sharded mongodb, by either setting
the MONGODB_SHARD_COLLECTIONS env variable, or bind-mouting the backbeat
or cloudserver config file.

Issue: S3UTILS-76

francoisferrand committed:
Use createMongoParams() in stalled script

This way, the script can be used with sharded mongodb, by either setting
the MONGODB_SHARD_COLLECTIONS env variable, or bind-mouting the backbeat
or cloudserver config file.

Issue: S3UTILS-76

bert-e commented:
# Integration data created



I have created the integration data for the additional destination branches.

* this pull request will merge `bugfix/S3UTILS-76` into
`development/1.12`
* [w/1.13/bugfix/S3UTILS-76](https://github.com/scality/s3utils/compare/development/1.13...w/1.13/bugfix/S3UTILS-76) will be merged into `development/1.13` (pull request #234)





*Follow* integration pull requests if you would like to be notified of
build statuses by email.

bert-e commented:
# Waiting for approval



The following approvals are needed before I can proceed with the merge:


* the author


* 2 peers

francoisferrand commented:
/approve

bert-e commented:
# In the queue



The changeset has received all authorizations and has been added to the
relevant queue(s). The queue(s) will be merged in the target development
branch(es) as soon as builds have passed.

The changeset will be merged in:


* :heavy_check_mark: `development/1.12`

* :heavy_check_mark: `development/1.13`




There is no action required on your side. You will be notified here once
the changeset has been merged. In the unlikely event that the changeset
fails permanently on the queue, a member of the admin team will
contact you to help resolve the matter.

*IMPORTANT*

Please do not attempt to modify this pull request.

* Any commit you add on the source branch will trigger a new cycle after the
  current queue is merged.
* Any commit you add on one of the integration branches will be **lost**.

If you need this pull request to be removed from the queue, please contact a
member of the admin team now.




*The following options are set:* **approve**

bert-e commented:
# 


I have successfully merged the changeset of this pull request
into targetted development branches:


* :heavy_check_mark: `development/1.12`

* :heavy_check_mark: `development/1.13`




Please check the status of the associated issue S3UTILS-76.

Goodbye francoisferrand.
